libsolidity/formal/CHC.cpp

```diff
@@ -156,19 +158,21 @@ void CHC::endVisit(ContractDefinition const& _contract)
 				auto const& params = baseConstructor->parameters();
 				solAssert(params.size() == args.size(), "");
 				for (unsigned i = 0; i < params.size(); ++i)
+				{
+					args.at(i)->accept(*this);
```

Comment:
@mijovic this is the fix for unnamed constructor parameters: the argument must be evaluated anyway, because of side effects.